pkg/kube/client.go

```diff
@@ -532,6 +532,7 @@ func (c *Client) WatchUntilReady(resources ResourceList, timeout time.Duration)
 }
 
 func perform(infos ResourceList, fn func(*resource.Info) error) error {
+	var lastError error
```

Comment:
Did you think about using a multi-error to capture all of them and return all of them?

Reply:
yeah , I searched two packages 
1. https://github.com/uber-go/multierr
2. https://github.com/hashicorp/go-multierror
But both of them are not active recently, and I am not sure which is better.   Do you have any suggestion to use which package ? or is there a new famous package ?

Reply:
@mattfarina  please review again